### PR TITLE
Handle boolean props in the Map Template

### PR DESCRIPTION
**Pull request by andreeaceachir142**
Handle boolean props in the Map Template by adding a new function `getBooleanValue`.
Add default values to all the boolean props and write unit tests for that.

### CODE REVIEW

**Review by ammapspeople — APPROVED** [review]

File: packages/map-template/package.json [hunk 1]
```diff
@@ -40,14 +40,15 @@
     "vite-plugin-eslint": "^1.8.1",
     "vite-plugin-favicon2": "^1.1.5",
     "vite-plugin-svgr": "^3.2.0",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4", 
+    "jest": "^29.7.0"
```
Comment (ammapspeople) [inline packages/map-template/package.json:44]: Let's keep them sorted alphabetically 😁

Comment (andreeaceachir142) [reply packages/map-template/package.json:44]: :404annesurprisedemojinotfound:

Comment (andreeaceachir142) [reply packages/map-template/package.json:44]: I did not know we sorted them alphabetically until now 😂

Comment (ammapspeople) [reply packages/map-template/package.json:44]: I guess that's also something for the contribution guide 🙃

Comment (andreeaceachir142) [reply packages/map-template/package.json]: Done ✅

**Comment by ammapspeople** [comment]
Re-approved 🆗 🆗 🆗

**Pull request merged**